corehq/apps/export/dbaccessors.py

```diff
@@ -0,0 +1,32 @@
+
+
+def get_latest_case_export_schema_id(domain, case_type):
+    from .models import ExportDataSchema
+
+    key = [domain, 'CaseExportDataSchema', case_type]
+    result = ExportDataSchema.get_db().view(
+        'schemas_by_xmlns_or_case_type/view',
+        startkey=key + [{}],
+        endkey=key,
+        include_docs=False,
+        limit=1,
+        reduce=False,
+        descending=True,
```

Comment:
I think you'd have to check for new builds every time otherwise the user wouldn't be able to add any new properties.

But :+1: that it only creates a new one if there are new builds.

I don't have very strong opinions about this but it does seem a bit wasteful to always be creating a new doc. What if you just give it the same `_id` and `_rev` as the old one so that it overwrites? Then we don't end up with a bunch of dangling schemas.